todo/style.css

```diff
@@ -1,354 +1,326 @@
+body {
+  background: #2f80ed;
+}
 
-body
-{
-    background: #2F80ED;
-    
+h1 {
+  font-family: "Sinhala MN";
+  font-style: normal;
+  font-weight: 400;
+  font-size: 50px;
+  line-height: 59px;
+  letter-spacing: 2px;
+  color: #ffffff;
 }
-.title
-{
-    font-family: 'Sinhala MN';
-    font-style: normal;
-    font-weight: 400;
-    font-size: 50px;
-    line-height: 59px;
-    letter-spacing: 0.1em;
-    color: #FFFFFF;
 
+h1 {
+  text-align: center;
+}
 
+.check-label {
+  display: block;
 }
-h1
-{
-    text-align: center;
+
+.text-todo {
+  text-align: left;
+  width: 604px;
+  height: 44px;
+  left: 0;
+  top: 0;
+  background: #ffffff;
+  outline: none;
+  font-family: "Roboto";
+  font-size: 15px;
+  border: 0;
```

Comment:
Please check styles order.
cursor - the first
then position/top/left etc.
then display/width/height etc
I've sent a link in our chat.